packages/gatsby-transformer-video/src/ffmpeg.js

```diff
@@ -0,0 +1,443 @@
+import { tmpdir } from "os"
+import { resolve, parse } from "path"
+
+import { createContentDigest } from "gatsby-core-utils"
+import { createRemoteFileNode } from "gatsby-source-filesystem"
+import { pathExists } from "fs-extra"
+import ffmpeg from "fluent-ffmpeg"
+import imagemin from "imagemin"
+import imageminGiflossy from "imagemin-giflossy"
+import PQueue from "p-queue"
+
+export default class FFMPEG {
+  constructor({ rootDir, ffmpegPath, ffprobePath }) {
+    this.queue = new PQueue({ concurrency: 1 })
+    this.cacheDir = resolve(
+      `${rootDir}/node_modules/.cache/gatsby-transformer-video/`
```

Comment:
a) should the cache of user data better somewhere in `www/.cache/gatsby-transformer-video/`?

Reply:
b) i can not see that `cacheDir` is used anymore?

Reply:
I readded this caching layer and moved it into `.cache`

Curious what gatsby cloud now will do, before this change it had to reconvert all videos on new branch builds